vagrant/provisioning/arkcase-ce.yml

```diff
@@ -13,6 +13,8 @@
       tags: [core, ldap]
     - role: mariadb
       tags: [core, mariadb]
+    - role: postgresql
+      tags: [core, postgresql]
```

Comment:
I wouldn't add this one as core role. By default we want mariadb.

Reply:
The `when` clause skips this role when the `db_engine` is not `mariadb`.  I have tested this; it works well.